tests/unit/utils.js

```diff
@@ -0,0 +1,60 @@
+ const assert = require('chai').assert;
+
+// create a global window so we can unittest the window.<x> functions
+if (!global.window)
+   global.window = new Object({});
+
+require('../../saltgui/static/scripts/utils');
+
```

Comment:
Is there a reason for two lines over one here?

Reply:
No, so I adjusted it.